services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/DisabledPartitionStats.java

```diff
@@ -3,19 +3,22 @@
 import com.linkedin.venice.stats.AbstractVeniceStats;
 import io.tehuti.metrics.MetricsRepository;
 import io.tehuti.metrics.Sensor;
-import io.tehuti.metrics.stats.Count;
+import io.tehuti.metrics.stats.Total;
 
 
 public class DisabledPartitionStats extends AbstractVeniceStats {
   private final Sensor disabledPartitionCount;
 
   public DisabledPartitionStats(MetricsRepository metricsRepository, String name) {
     super(metricsRepository, name);
-    disabledPartitionCount = registerSensorIfAbsent("disabled_partition_count", new Count());
+    disabledPartitionCount = registerSensorIfAbsent("disabled_partition_count", new Total());
```

Comment:
In that case, I believe the disabled metric value is no longer valid, right? It could contain false information.

Reply:
Yes, but we will be able to see how many leader replicas are getting disabled over periods of time in different clusters/stores. let's say if the disabled count is 100 and the reenabled count is also 100, with the current change it will only show 0, which is what we are looking for, but it doesn't say that there were 100 replicas that were disabled in the first place. Logs will show this but might be good to have a metric to see if any particular store/cluster is producing a lot of disabled replicas. this was my intention behind this comment.